components/switch/switch-mixin.js

```diff
@@ -68,6 +68,9 @@ export const SwitchMixin = superclass => class extends RtlMixin(FocusVisiblePoly
 			:host([on]) .d2l-switch-inner {
 				background-color: var(--d2l-color-celestine-plus-2);
 			}
+			:host(:not([on])) .d2l-switch-inner {
```

Comment:
Could this be moved up to the `.d2l-switch-inner` selector (i.e. to avoid the `[on]`, `:not()`, and `:host` selector checks)?